README.md

```diff
@@ -88,6 +88,16 @@ includes builds of this Go fork.
 * In Azure Linux 2.0, the package `msft-golang` installs this fork.
 * In Azure Linux 3.0, the `golang` package installs this fork.
 
+### Ubuntu
+
+To install the Microsoft build of Go using an Ubuntu package, first set up the [Linux package repository for Microsoft Products](https://learn.microsoft.com/en-us/linux/packages).
+Packages are available in the Ubuntu 22.04 and 24.04 repositories.
+After the repository is added, install the Microsoft build of Go by running the following commands:
```

Comment:
```suggestion

After the repository is added, install the Microsoft build of Go by running the following commands:
```

Missed newline. (So that those who have the repo already set up can easily visually scan to the next step.)